flatliners/baseflatliner.py

```diff
@@ -38,3 +38,6 @@ def metric_name(self, x):
 
     def cluster_id(self, x):
         return self.metric_label(x, '_id')
+
+    def alert_name(self, x):
+        return self.metric_label(x, 'alertname')
```

Comment:
This should be in your `dfgen`
It's not useful to the other flatliners

Reply:
I agree, will remove